src/pages/cart/cart.css

```diff
@@ -69,4 +69,9 @@
 
 .space-between-cart{
     justify-content: space-between;
+}
+
+
+.quantity{
+    font-size: 1.5rem;
```

Comment:
its good to use font-size in em's unit rather than rems

Reply:
ok